lib/diary.scss

```diff
@@ -40,7 +40,7 @@
   height: 25px;
   align-items: center;
   justify-content: space-between;
-  width: 250px;
+  width: 248px;
```

Comment:
Very good touch to align the rows and add consistency throughout all the tables.